base/bucket_gocb.go

```diff
@@ -917,13 +923,13 @@ func (bucket CouchbaseBucketGoCB) GetWithXattr(k string, xattrKey string, rv int
 			// Successfully retrieved doc and (optionally) xattr.  Copy the contents into rv, xv and return.
 			contentErr := res.Content("", rv)
 			if contentErr != nil {
-				LogTo("gocb", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
-				return false, contentErr, nil
+				LogTo("CRUD", "Unable to retrieve document content for key=%s, xattrKey=%s: %v", k, xattrKey, contentErr)
+				return false, contentErr, 0
```

Comment:
Ok that's surprising .. looking at the usage, I would have expected the RetryLoop to return whatever was returned from the `worker` callback.

Ok, now at least the code makes a little more sense.

I can see how it would work, but wouldn't this be clearer? 

```
if err != nil {
    return 0, err
}
```